core/modules/menu/menu.admin.inc

```diff
@@ -423,14 +453,22 @@ function menu_edit_item($form, &$form_state, $type, $item, $menu) {
     '#title' => t('Parent link'),
     '#default_value' => $default,
     '#options' => $options,
-    '#description' => t('The maximum depth for a link and all its children is fixed at !maxdepth. Some menu links may not be available as parents if selecting them would exceed this limit.', array('!maxdepth' => MENU_MAX_DEPTH)),
+    '#description' => t('The maximum depth for a link and all its children is fixed at !maxdepth. Some menu links may not be available as parents if selecting them would exceed this limit. <a href="!url">Configure this selector.</a>', array(
+      '!maxdepth' => MENU_MAX_DEPTH,
+      '!url' => url('admin/structure/menu/settings'),
+    )),
```

Comment:
We should be conditionally adding the link here, after doing a `user_access()` to make sure that the current user has permission to configure the menu settings.

Reply:
At any rate, since I've now removed the setting, this link is no longer present.